server/src/server.ts

```diff
@@ -28,7 +28,7 @@ let activeLanguages: string[];
 // A list of file extensions to lookup for style definitions (defaults to .css, .scss and .less)
 let fileSearchExtensions: string[]
 
-documents.onDidOpen((event) => {
+let onFileOpen = (event: any) => {
```

Comment:
```javascript
const onFileOpenOrChange = event => {
```

Also, don't use `any`. This event is already implicitly of type `TextDocumentChangeEvent` so there's no reason to make the type more generic when we already know what they type is